code/game/machinery/bank_machine.dm

```diff
@@ -0,0 +1,78 @@
+/obj/machinery/computer/bank_machine
+	name = "bank machine"
+	desc = "A machine used to deposit and withdraw station funds."
+	icon = 'goon/icons/obj/goon_terminals.dmi'
+	icon_state = "atm"
+//	req_access = list(access_quartermaster) //reqaccess to access all stored items
+	anchored = 1
+	use_power = 1
+	idle_power_usage = 100
+	var/siphoning = FALSE
+	var/last_warning = 0
+
+/obj/machinery/computer/bank_machine/attackby(obj/item/I, mob/user)
+	var/value = 0
+	if(istype(I, /obj/item/stack/spacecash))
+		var/obj/item/stack/spacecash/C = I
+		value = C.value
+	if(istype(I, /obj/item/weapon/coin))
+		var/obj/item/weapon/coin/C  = I
+		value = C.value
+	if(value)
+		SSshuttle.points += value
+		user << "<span class='notice'>You deposit [I]. The station now has [SSshuttle.points] credits.</span>"
+		qdel(I)
+		return
+	return ..()
+
+
+/obj/machinery/computer/bank_machine/update_icon()
```

Comment:
I was about to do this and then saw that the screens are animated and directional and I cried